CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ A new [**facetAnchor**](#facetanchor) option, available for all marks, controls
 
 A new **anchor** option allows to draw only one side of the [frame](./README.md#frame) mark—in which case the **fill**, **fillOpacity**, **rx**, and **ry** options are ignored.
 
+The new [auto mark](./README.md#auto) automatically selects a mark type that best represents the dimensions of the given data according to some simple heuristics. For example, `Plot.auto(olympians, {x: "height", y: "weight"}).plot()` makes a scatterplot; `Plot.auto(aapl, {x: "Date", y: "Close"}).plot()` makes a line chart; `Plot.auto(penguins, {x: "body_mass_g"}).plot()` makes a histogram; `Plot.auto(penguins, {x: "island"}).plot()` makes a bar chart.
```

Comment:
"best" (TK)

Reply:
Deferring to @mbostock comments in docs notebook, I'll leave this…?

Reply:
yup